paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodEmbeddedLayoutUIScreenshotTest.kt

```diff
@@ -148,7 +152,8 @@ class PaymentMethodEmbeddedLayoutUIScreenshotTest {
             onSelectSavedPaymentMethod = {},
             onManageOneSavedPaymentMethod = {},
             imageLoader = mock(),
-            rowStyle = rowStyle
+            rowStyle = rowStyle,
+            modifier = Modifier.verticalScroll(scrollState),
```

Comment:
The `fillMaxHeight` causes overly aggressive height if not in a scrollable container.